spec/exp_descriptor_nl_spec.rb

```diff
@@ -0,0 +1,431 @@
+# encoding: utf-8
```

Comment:
use a non-english version of the spec as the basis. each of them should have 405 lines

Reply:
I made the change by removing `strict_quartz` and `timezone` to end up on 405 lines.